public/assets/meetups/2024-09-11-lmt-02/data.json

```diff
@@ -0,0 +1,98 @@
+{
+  "title": "II legnicki meetup technologiczny",
+  "datetime": "2024-09-11 18:00:00",
```

Comment:
According to lmt Facebook page, the meetup's date is 2024-09-12

Reply:
The data I provided is just for testing. File with correct data should be added before the next meetup.